policies/README.md

```diff
@@ -0,0 +1,84 @@
+
+## OPA POLICIES
+
+Policies are rules that can be applied when any of your cluster object is _created_, _updated_, _patched_ and _deleted_.
+
+Policies are always executed _before_ the action takes place and their goal is to allow or prevent the modification.
+
+This solution is very efficient in term of required computing time thanks to the OPA gatekeeper.
+
+The language used by OPA gatekeeper is [REGO] (https://www.openpolicyagent.org/docs/latest/policy-language/).
+
+### HOW TO USE
+
+To use OPA gatekeeper you need to:
+
+- Deploy a Release using Prebuilt Image to create in your cluster the custom objects needed for gatekeeper to work. See [link](../infrastructure/policy-enforcement/README.md).
+
+- Create your own constraint and template files and eventually, if needed, config-sync file. See [examples](../infrastructure/policy-enforcement/examples).
+
+Now you can test your policies on your cluster objects.
+
+### HOW TO USE GATEKEEPER
+
+### Template
+
+In the Template you should define your own constraint CRD and associate it to your rego policies, such as in this example: [**template.yaml**](../infrastructure/policy-enforcement/examples/template.yaml)
+
+### Constraint
+
+In Constraint you should declare the resources on which your Template operates and define possible optional parameters (e.g., reject a modification if a value is greater than a given threshold), for example: **[constraint.yaml](../infrastructure/policy-enforcement/examples/constraint.yaml)**
+You can retrieve your input object using `input` variable.
+In OPA, input is a reserved, global variable whose value is the Kubernetes AdmissionReview object that the API server hands to any admission control webhook.
+
+### Config-Sync
+
+Some constraints are impossible to write without access to more state than just the object under test. For example, it is impossible to know if an instance is valid unless a rule has access to all the templates. To make such rules possible, we enable syncing of data into OPA.
```

Comment:
Again, a reference should be added if you copy text from other places, such as the official documentation.